routes/documents.js

```diff
@@ -3,11 +3,29 @@ const router = express.Router();
 
 // Import classes 
 const { File, DocumentType } = require('../classes/File.js');
+const { MediaFile } = require('../classes/MediaFile.js');
 const { 
   attachReadRouteHandlerWrapper, 
   attachWriteRouteHandlerWrapper
 } = require('../middleware/routeHandler')
 
+const extractDirectoryAndFileName = (documentName) => {
+  let documentDirectory, documentFileName
+
+  const pathArr = documentName.split('/')
+  if (pathArr.length === 1) {
+    documentDirectory = 'files'
+    documentFileName = documentName
+  } else if (pathArr.length > 1) {
+    documentDirectory = `files/${pathArr.slice(0, -1)}`
```

Comment:
nit: can we leave a short comment what `pathArr` contains and why we need `pathArr.slice(0,-1)`

Reply:
Added comments in https://github.com/isomerpages/isomercms-backend/pull/169/commits/bf0a6684e8af0543ddc2cde08a9f068b6e0c5261